app/helpers/media_helper.rb

```diff
@@ -1,13 +1,11 @@
 module MediaHelper
   def media_thumbnail(media_item)
-    if media_item.thumbnail?
-      if media_item.caption.text.present?
-        alt_text = truncate(media_item.caption.text, length: 36)
-        return image_tag(media_item.item.thumb.url, alt: alt_text)
-      else
-        # Use auto-generated alt text
-        return image_tag(media_item.item.thumb.url)
-      end
+    if media_item.thumbnail? && media_item.caption
```

Comment:
I'd take out `media_item.caption` here. We still want to account for any cases, if they exist, where there is no caption tied to a thumbnail.

It looks like the reason the loan that is broken is failing is because there exists an image that does not have a caption tied to it at all.

Reply:
👍